config/settings/development.yml

```diff
@@ -27,3 +27,12 @@ web_origin_regex: \Ahttps?:\/\/.*\z
 modules_appeals_api:
   documentation:
     notice_of_disagreements_v1: true
+
```

Comment:
Please add this to config/settings.yml.

Reply:
To clarify what Rachal is asking here, `vets-api` doesn't really use the standard `development.yml`/`production.yml`, etc way of setting stack specific settings. Localhost settings need to be in `config/settings.yml` as Rachal asked, and then settings for other stacks need to be set in this repository: https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests

Reply:
updated as requested

Reply:
@binq Please remove the changes to this file now that they're in config/settings.yml.

Reply:
done